ATParser.h

```diff
@@ -72,6 +74,28 @@ class ATParser
     * @param serial serial interface to use for AT commands
     * @param buffer_size size of internal buffer for transaction
     * @param timeout timeout of the connection
+    * @param send_delimiter string of characters to use as line delimiters for sending
+    * @param recv_delimiter string of characters to use as line delimiters for receiving
+    */
+    ATParser(BufferedSerial &serial, const char *recv_delimiter = "\r\n", const char *send_delimiter = NULL, int buffer_size = 256, int timeout = 8000, bool debug = false) :
```

Comment:
I think for this to work you need to actually omit the default arguments for recv_delimiter and send_delimiter, otherwise declarations like `ATParser at(&serial)` are ambiguous.